### ISSUE TITLE
Connecting to a Database Using Port Forwarding

### ISSUE THREAD

**Issue by venphil** [openshift-labs/learn-katacoda#191]
In Getting Started with OpenShift for Developers::Connecting to a Database Using Port Forwarding:

entering the command
   oc new-app postgresql-ephemeral ...

results in an error -- no match for "postgresql-ephemeral".

Is there an updated version of this tutorial, or something else I need to do?

Thanks,
Philip Baker
venphil@gmail.com

**Comment by GrahamDumpleton** [comment]
Can you try again?

It may have been a transitory issue with the environment. It is working now when I test it.

If you still have an issue, can you click on the icon to the right of the 'Dashboard' tab which looks like a square box with arrow pointing up to the right. That will bring up the dashboard in a separate tab/window. What hostname does that show, eg:

```
2886795284-8443-simba02.environments
```

I am running on ``simba02``, but there are other instances that are used. If know which environment we can see if it is having an issue. Thanks.

**Comment by venphil** [comment]
Still having the problem.  Here is the host name per your instructions:

https://2886795363-8443-simba02.environments.katacoda.com/login?then=%...

BTW, I', running Mac OS X High Sierra.

**Comment by venphil** [comment]
Another problem, this time in  Topic 2 - Downloading Files from a Container.

~ $ oc new-app openshiftkatacoda/blog-django-py --name blog

error: yaml: line 7: mapping values are not allowed in this context

It must be the case that I'm missing a crucial component, but I don't know
what it is. Something to do with *git* maybe?

--Phil Baker

**Comment by GrahamDumpleton** [comment]
I have tried both scenarios again and they work as intended and nothing with them has changed for months, so any issue can only be with the environments.

You must be just unlucky, or trying to do it at a time when the system is quite loaded. With OpenShift Gathering and Kubecon conference on at the moment, it will be getting a lot of use and that may cause things to be slow. Some steps in the setup are done in parallel to you reading instructions, so especially for the PostgreSQL template, if the system is slow and you rush through the steps, there is a small chance the template may not have been loaded by that time. I can't explain the YAML error though.

Are you clicking on the commands in the instructions so they are automatically run for you in the terminal window to the right, or are you entering themselves in the terminal window in the browser?

**Comment by GrahamDumpleton** [comment]
Going through the scenario for copying files again, have seen this time an issue, but it is past where you were. It is an issue with persistent volumes not being set up correctly. So suggests there is some issue perhaps with sequence of things and how long they take at startup. I will see if can work out what is happening with that one at least.

**Comment by GrahamDumpleton** [comment]
Running again and it works. The problem appears to be the speed at which images are being pulled from Docker Hub, or errors in pulling the images. The setup of the persistent volumes is only done afterwards and if the pulling of the image fails, it aborts the setup and subsequent steps of creating persistent volumes never occurs. I have seen comments about recent issues with pulling images from Docker Hub due to problems with Docker Hub, so maybe is related to that.

**Comment by jankleinert** [comment]
Closing this since there haven't been any updates in over 1 year. Please open a new issue if this persists.
